_posts/2016-12-12-Webinar-on-Salesforce-Connect-External-Object-Reporting.md

```diff
@@ -17,21 +17,21 @@ For a developer preview, below are screenshots of the [high level steps from the
 
 *1. Get started with an External Object Report*
 
-<img src="/assets/SalesforceWebinar1.png" width="623" height="348" />
+<img src="/assets/SalesforceWebinar1.png" alt="External Object Report" width="623" height="348" />
```

Comment:
Maybe give more information in the alt so that a screen reader user can get better information. Tabbing through the images with narrator would probably make someone think they are stuck on the same element.

Reply:
updated